Example/AltSwiftUIExample/ViewController.swift

```diff
@@ -8,8 +8,8 @@
 
 import UIKit
 import AltSwiftUI
-import protocol AltSwiftUI.ObservableObject
-import class AltSwiftUI.Published
+import protocol AltSwiftUI.ObservableObject // swiftlint:disable:current duplicate_imports
+import class AltSwiftUI.Published // swiftlint:disable:current duplicate_imports
```

Comment:
It may be better to disable this rule altogether as specific imports are a valid way to disambiguate but swift lint doesn't handle it properly.

Reply:
Disabled